README.md

```
@@ -10,9 +10,9 @@ Split-Transformer with Integrated Convolutions for Genotype Imputation source co
   <img width="100%" height="auto" src="https://github.com/shilab/STICI/blob/d0da6cddeb071ef41117936467e2297a999de286/architecture.png">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)